app/src/main/res/layout/app_bar_main.xml

```diff
@@ -25,6 +25,7 @@
         android:clickable="true"
         android:focusable="true"
         android:text="@string/ask_question"
+        android:visibility="gone"
```

Comment:
Is this an argument of the ask question button? This button shouldn't be here.

Reply:
done